### PR TITLE
rpc: update `getversion` RPC-call

### LINKED ISSUE #1035: Fix getversion RPC call for 3.0
https://docs.neo.org/v3/docs/en-us/reference/rpc/latest-version/api/getversion.html

The answer format is a bit different now.

### PR DESCRIPTION
**Pull request by AnnaShaleva**
closes #1035

### CODE REVIEW

File: pkg/rpc/response/result/version.go [hunk 1]
```diff
@@ -4,7 +4,7 @@ type (
 	// Version model used for reporting server version
 	// info.
 	Version struct {
-		Port      uint16 `json:"tcp_port"`
+		Port      uint16 `json:"tcpPort"`
 		Nonce     uint32 `json:"nonce"`
 		UserAgent string `json:"useragent"`
```
Comment (roman-khimov) [inline pkg/rpc/response/result/version.go:9]: It's `user_agent` now.

File: pkg/rpc/response/result/version.go [hunk 2]
```diff
@@ -4,7 +4,7 @@ type (
 	// Version model used for reporting server version
 	// info.
 	Version struct {
-		Port      uint16 `json:"tcp_port"`
+		Port      uint16 `json:"tcpPort"`
```
Comment (roman-khimov) [inline pkg/rpc/response/result/version.go:7]: I don't think it's enough, this structure needs to be compatible with all nodes and some will send us `ws_port` that we should be able to represent correctly on the client (even though our server never sends it).

Comment (roman-khimov) [reply pkg/rpc/response/result/version.go:7]: And where this `tcpPort` comes from? I see `tcp_port` in the `master` code, am I missing something?

Comment (AnnaShaleva) [reply pkg/rpc/response/result/version.go]: `tcpPort` and `useragent` were taken from specification (https://docs.neo.org/v3/docs/en-us/reference/rpc/latest-version/api/getversion.html). Now replaced by `tcp_port` and `user_agent`.

Comment (roman-khimov) [reply pkg/rpc/response/result/version.go]: Wow. So there is a difference between implementation and documentation? Please file a bug for neo-modules then. We'll use implementation's field names for now.

**Pull request merged**